js/languages/fr.js

```diff
@@ -0,0 +1,326 @@
+    'create-assync': 'Créer un compte Assync',
+    'or-assync': ' ou ',
+    'join-assync': 'Synchroniser avec le compte',
+    'leave-assync': 'Arrêtez la synchronisation',
+    'assync-id': 'Votre identifiant de compte Assync: ',
+    'loading-assync': 'Chargement...',
+    errors: "Journal des erreurs d'exécution",
+    'edit-h': 'Réglez les heures de la période H',
+    'h-editor': 'Paramètres de période H',
+    'toggle-pd-add-asgn': 'Afficher le bouton "Ajouter un devoir" sur les fiches de période?',
+    support: 'Ressources étudiantes',
+    sponsor: 'Parrainé par le ROCK team.',
+    'show-self-toggle': 'Afficher la période SELF?',
+    show0: 'Afficher la période zéro?',
+    s1a: 'Ligne crise de textos',
+    s1b: '741-741',
+    s1c: 'Pour tout le monde en crise; texte "Help"\nTexte "LGBTQ" pour l\'aider spécifique aux LGBTQQ',
+    s2a: 'Hotline de prévention du suicide',
+    s2b: '800-273-8256',
+    s2c: 'Ressources de prévention et de crise (nationales)',
+    s3a: 'Ligne suicide et crise',
+    s3b: '855-278-4204',
+    s3c: 'Pour les personnes en crise (Comté de Santa Clara)',
+    s4a: 'Star Vista',
+    s4b: '650-579-0350',
+    s4c: 'Intervention de crise (Comté de San Mateo)',
+    s5a: 'Uplift',
+    s5b: '408-379-9085',
+    s5c: 'Intervention mobile en cas de crise et planification de la sécurité',
+    s6a: 'Trevor Lifeline',
+    s6b: '866-488-7386',
+    s6c: 'Intervention de crise LGBTQ et prévention du suicide',
+    'staff-disclaimer-before-link': "Veuillez noter que les informations sur le personnel ont été extraites du ",
+    'staff-disclaimer-link': "répertoire du personnel de l'administration en date du 11 Septembre 2019",
+    'staff-disclaimer-after-link': ', alors attribuez-leur des inexactitudes.',
+    announcements: 'Annonces',
+    loading: 'Chargement...',
+    new: 'NOUVEAU',
+    'hide-preps': 'Masquer les périodes avec "prep" dans leurs noms dans l\'horaire?',
+    'martin-ad': 'Site Web de calcul GPA de Martin',
+    ad: 'Utilité convaincante'
+  },
+  other: {
+    'anti-ugwaga': 'Cliquez / tapez pour continuer vers le Unofficial Gunn Web App',
+    'no-study-before-emph': 'Tu ',
+    'no-study-emph': "n'as pas besoin d'étudier",
+    'no-study-after-emph': "; même si vous marquez 0%, vous serez au-dessus de votre seuil.",
+    'zero-error': "Veuillez ne pas entrer autant de zéros.",
+    'minscore-before-emph': "Vous devrez marquer au moins ",
+    'minscore-after-emph': ' pour garder vos parents heureux.',
+    'minscore-too-high-addendum': " S'il n'y a pas de crédit supplémentaire, vous êtes foutu.",
+    gmaps: 'utiliser google maps',
+    image: "utiliser l'image",
+    'gmaps-error': "Google Maps ne se charge pas! Peut-être que vous n'êtes pas connecté à Internet?",
+    you: 'Vous',
+    'barcode-legacy-default': 'Étudiant #{N}',
+    'barcode-default': 'Étudiant intellectuel',
+    'barcode-student-placeholder': "A qui est cette pièce d'identité?",
+    ds: 'D  L  M  ᗰ  J  V  S',

```

Comment:
?

Reply:
The letters for the days of the week. In English it'd be `S  M  T  W  Θ  F  S`. It uses two spaces as a separator just in case a letter needs a space? and there's a fancy M to distinguish martes from miercoles (in French) like how I used T and Θ for Tuesday and Thursday